src/types/index.ts

```diff
@@ -14,7 +14,6 @@ type TransformedEvent = {
   };
   files?: Record<string, unknown>;
   metadata?: Metadata;
-  [key: string]: any;
 };
 
 type Metadata = {
```

Comment:
From all the type and field names, replace `transform` with `transformation`, as the former is a verb and the latter is a noun.

Reply:
Addressed